sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationClientOptions.cs

```diff
@@ -31,6 +31,9 @@ public DocumentTranslationClientOptions(ServiceVersion version = LatestVersion)
         {
             Version = version;
             AddLoggedHeadersAndQueryParameters();
+
+            //Default Audience to Azure Public Cloud
+            Audience ??= DocumentTranslationAudience.AzurePublicCloud;
```

Comment:
I know this probably comes from the implementation in other services, but this looks a bit weird to me.

Firstly, `Audience` is guaranteed to be null here since it has never been initialized, so the null check is not doing anything.

Secondly, in the `Audience` property docstrings, we state that we'll assume public cloud if null, so my guess is that we intended to keep it null and not initialize it in the constructor.

It seems that in TA we keep it null while in FR we didn't. They have not GA-ed yet, so we still have time to unify.

Reply:
Actually this pattern is present across SDKs not just cognitive space. For example ACR also has this.

@maririos I agree with @kinelski here. We can let this be null. we already have code in the Client to default to AzureCloud instead. Options values should be true to what the customer chose. 

Let me know if we did this based on the archboard feedback, in which case we should potentially change other SDK implementations too.

Reply:
Just found the original discussion in the FR PR: https://github.com/Azure/azure-sdk-for-net/pull/26741#discussion_r797788186

Per Jesse's comment:
> We want the options to default to null; the client is applying the default.

I think we forgot to update the code in some places. I'll open a GH issue to track the work in FR and we can align with TA (default `Audience` to null). What do you think?

Reply:
I agree. @maririos let us know if you think otherwise

Reply:
Thank you for the context! I will update

Reply:
Updated